samcli/lib/replace_uri/replace_uri.py

```diff
@@ -0,0 +1,100 @@
+from samcli.lib.utils.packagetype import ZIP
+
+
+
+
+class ReplaceLocalCodeUri:
```

Comment:
Great job extracting this into separate file. Looking at the class, there is only one instance with single method with no parameters. I believe this could be a single function instead of a class. Then you can directly call this from the `sam check` command as well.

Reply:
Plus I didn't see unit tests for this new file. When you extract these please don't forget to update existing tests + add new ones to cover new lines.

Reply:
The code for the class ReplaceLocalCodeUri (I will change it to a single function) is pulled from sam_template_validator. There is already test code for that, as the code previously existed there. Those tests test the code in the class ReplaceLocalCodeUri. If I were to write tests for the class ReplaceLocalCodeUri, they would be 100% identical to the tests for sam_template_validator, as that is where the code use to exist. Even though the tests would be 100% identical, is it still proper to have them written in their own file? It seems unnecessary to have the same code written in two different places. Should I instead "point" the current tests for sam_template_validator to instead test the class ReplaceLocalCodeUri directly, since that is where the logic is now found?